data/tags.yaml

```
@@ -18,3 +18,30 @@
 #        create_view: true
 #        view_name: 'TEST view name'
 # ------------------------------------------------------------------------
+gcs_bucket_creation_tagger:
+  query_string: 'serviceName: "storage.googleapis.com" AND methodName: "storage.buckets.create"'
+  tags: ['gcs_bucket_create']
```

Lets use dashes instead:

```suggestion
  tags: ['gcs-bucket-create']
```